packages/sakuli-legacy/src/command/create-commands/create-project-command.class.ts

```diff
@@ -0,0 +1,21 @@
+import {Argv} from "yargs";
+import {createTestsuite} from "../../init-command/createStructure";
+
+export = {
+    command: 'project [directory] [suiteName]',
+    describe: 'Generates a default project structure',
+    builder(argv: Argv) {
+        return argv.positional('directory', {
+            describe: 'path where the sakuli project should be created'
+            }).demandOption('directory')
+            .positional('suiteName', {
+                describe: 'name of the testsuite',
+                default: 'sakuli'
+            });
+    },
+    async handler(opts: any) {
+        console.log("creating project structure");
+        createTestsuite(opts.directory, opts.suiteName);
+        process.exit(0);
```

Comment:
We should provide more details to the user here:
- `creating project in $absolutePathInUserFileSystem`
- `successfully create testsuite`
and so on.

Additionally we should check if `opts.directory` is a sakuli project root otherwise we should warn the user. We can also think about creating a proper package.json file and install sakuli there.

I actually think that the directory can also be optional with a default at the current workdirectory where the user runs the command (`process.pwd()`).

What do you think @s1hofmann  @svettwer

Reply:
test 2